Makefile.work

```diff
@@ -112,7 +112,7 @@ endif
 ifneq (,$(filter $(CONFIGURED_ARCH), armhf arm64))
     # Multiarch docker cannot start dockerd service due to iptables cannot run over different arch kernel
     SONIC_SERVICE_DOCKERD_FOR_MULTIARCH=y
-    SONIC_NATIVE_DOCKERD_FOR_MUTLIARCH := dockerd  --experimental=true --storage-driver=vfs \
+    SONIC_NATIVE_DOCKERD_FOR_MUTLIARCH := dockerd  --experimental=true --storage-driver=overlay2 \
```

Comment:
why swithc to overlay2, we discovered some compatibility issue for overlay2 before, therefore use vfs.

Reply:
VFS takes lot of space compared to overlay2 and also Makefile always check for overlay2 module in  build linux before starting build.
Also this docker daemon is for building purpose and not for creating the target container images.

Reply:
it should be consistent, suggest change overlay2 for all in another pr.

Reply:
Ok will revert overlay2 from this PR